scripts/publish.sh

```diff
@@ -47,17 +47,37 @@ popd
 npm ci
 npm run ts-build
 
-# pack utils and install it in shared
+# pack shared dependencies
 mkdir -p "$GIT_ROOT_DIR/packages/shared/vendor"
+mkdir -p "$GIT_ROOT_DIR/packages/database/vendor"
 pushd "$GIT_ROOT_DIR/packages/utils"
 jq '.bundleDependencies = true' package.json >temp.json && mv temp.json package.json
 npm install --workspaces=false
 npm pack --pack-destination "$GIT_ROOT_DIR/packages/shared/vendor"
+cp "$GIT_ROOT_DIR/packages/shared/vendor/nangohq-utils-1.0.0.tgz" "$GIT_ROOT_DIR/packages/database/vendor"
 popd
 pushd "$GIT_ROOT_DIR/packages/shared"
 npm install "@nangohq/utils@file:vendor/nangohq-utils-1.0.0.tgz" --workspaces=false
 popd
 
+pushd "$GIT_ROOT_DIR/packages/database"
+jq '.bundleDependencies = true' package.json >temp.json && mv temp.json package.json
+npm install "@nangohq/utils@file:vendor/nangohq-utils-1.0.0.tgz" --workspaces=false
+npm install --workspaces=false
+npm pack --pack-destination "$GIT_ROOT_DIR/packages/shared/vendor"
+popd
+pushd "$GIT_ROOT_DIR/packages/shared"
+npm install "@nangohq/database@file:vendor/nangohq-database-1.0.0.tgz"
```

Comment:
I feel like this double bundling will make the cli install even worst

Reply:
about 5 seconds slower

```
npm install @nangohq/cli@0.0.1-bf727d78a619ba3472a30392fa307cb8a424b214  11.30s user 4.37s system 110% cpu 14.121 total
```

```
npm install nango  6.55s user 3.20s system 139% cpu 6.982 total
```